enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackup.java

```diff
@@ -324,7 +324,7 @@ public OnlineBackup incremental( String targetDirectory )
     public OnlineBackup incremental( String targetDirectory, boolean verification )
     {
         outcome = new BackupService().doIncrementalBackup( hostNameOrIp, port, new File( targetDirectory ),
-                timeoutMillis, defaultConfig() );
+                getConsistencyCheck( verification ), timeoutMillis, defaultConfig() );
```

Comment:
So `verfication` wasn't quite used before? I'd guess it's fine to actually make use of it here, yeah.